### ISSUE TITLE
TS Error

### ISSUE THREAD

**Issue by zRelux** [roto93/react-native-neat-date-picker#32]
I upgraded from .9 to .12 and started getting this error:

```
node_modules/react-native-neat-date-picker/src/index.ts:3:15 - error TS2614: Module '"./components/NeatDatePicker"' has no exported member 'NeatDatePickerProps'. Did you mean to use 'import NeatDatePickerProps from "./components/NeatDatePicker"' instead?

3 import type { NeatDatePickerProps, ColorOptions } from './components/NeatDatePicker'
                ~~~~~~~~~~~~~~~~~~~

node_modules/react-native-neat-date-picker/src/index.ts:3:36 - error TS2614: Module '"./components/NeatDatePicker"' has no exported member 'ColorOptions'. Did you mean to use 'import ColorOptions from "./components/NeatDatePicker"' instead?

3 import type { NeatDatePickerProps, ColorOptions } from './components/NeatDatePicker'
                                     ~~~~~~~~~~~~


Found 2 errors in the same file, starting at: node_modules/react-native-neat-date-picker/src/index.ts:3
```

Looking at the code seems like TS is right any clue?

**Comment by diecodev** [comment]
Hi! Can you provide more info about your project? or create some example project to view the error.

**Comment by roto93** [comment]
I just publish a new version. I think this issue is solved in v1.4.13.
Feel free to reopen it if you still see the error.
